client/resources/aria-at-harness.mjs

```diff
@@ -114,12 +123,17 @@ export function verifyATBehavior(atBehavior) {
 }
 
 export async function loadCollectedTestAsync(testRoot, testFileName) {
-  const commandsJsonResponse = await fetch(`../commands.json`);
-  const commandsJson = await commandsJsonResponse.json();
-
   const collectedTestResponse = await fetch(`${testRoot}/${testFileName}`);
   const collectedTestJson = await collectedTestResponse.json();
-  testRunIO.setAllCommandsInputFromJSON(commandsJson);
+
+  try {
+    // v2 commands.json
+    const commandsJsonResponse = await fetch(`../commands.json`);
+    const commandsJson = await commandsJsonResponse.json();
+    testRunIO.setAllCommandsInputFromJSON(commandsJson);
+  } catch (e) {
+    // v2 commands.json isn't available
+  }
```

Comment:
This error handling is a little too permissive (it supports cases where the file is present but can't be read/decoded, for example). The risk is that future unexpected behavior will be obfuscated when the code quietly allows truly exceptional circumstances. What about something like this, instead?

```suggestion
  // v2 commands.json
  const commandsJsonResponse = await fetch(`../commands.json`);
  if (commandsJsonResponse.ok) {
    const commandsJson = await commandsJsonResponse.json();
    testRunIO.setAllCommandsInputFromJSON(commandsJson);
  }
```

Reply:
Keep forgetting how simple fetch makes that. Thanks! Done in https://github.com/w3c/aria-at/pull/1016/commits/57c20f846c383dae56afa2dd399e0114b8ee5f82, and the change will be moved over to here.